app/src/main/java/com/example/trempel/ui/pdp/PdpFragment.kt

```diff
@@ -31,7 +32,7 @@ internal class PdpFragment : Fragment() {
             this.lifecycleOwner = this@PdpFragment
             this.viewModel = this@PdpFragment.viewModel
         }.also {
-            viewModel.loadProduct()
+            viewModel.loadProduct(args.idProduct)
```

Comment:
I'd prefer naming like `productId` instead of `idProduct`

Reply:
done